packages/env-tests/src/tests/exchange.ts

```diff
@@ -2,73 +2,94 @@ import { sleep } from '@celo/base'
 import { describe, test } from '@jest/globals'
 import BigNumber from 'bignumber.js'
 import { EnvTestContext } from '../context'
-import { fundAccount, getKey, ONE, TestAccounts } from '../scaffold'
+import {
+  fundAccountWithStableTokens,
+  getKey,
+  initExchangeFromRegistry,
+  initStableTokenFromRegistry,
+  ONE,
+  TestAccounts,
+} from '../scaffold'
 
-export function runExchangeTest(context: EnvTestContext) {
+export function runExchangeTest(context: EnvTestContext, stableTokensToTest: string[]) {
   describe('Exchange Test', () => {
     const logger = context.logger.child({ test: 'exchange' })
     beforeAll(async () => {
-      await fundAccount(context, TestAccounts.Exchange, ONE.times(10))
+      await fundAccountWithStableTokens(
```

Comment:
Like the above comment, consider just having a test group per stable token than having a single test do multiple things

Reply:
Fully agree. Moved the funding inside the test for each stable token. This way only the stabletoken required for the current test running is funded. Plus cUSD or cEUR test will still run if funding accounts with cEUR or cUSD fails, respectively. Let me know your thoughts